lib/registrations/lanes/competing.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Registrations
+  module Lanes
+    module Competing
+      def self.process!(lane_params, user_id, competition_id)
+        registration = Registration.build(competition_id: competition_id,
+                                          user_id: user_id,
+                                          comments: lane_params[:competing][:comment] || '',
+                                          guests: lane_params[:guests] || 0)
```

Comment:
this is on purpose btw. currently guests is not part of the `[:competing]` payload right now (but probably will be in the future when API Design is finalized). I think our idea was that guests could be generalized for staff only registrations and such.